alphastats/gui/utils/overview_helper.py

```diff
@@ -21,47 +20,17 @@ def get_intensity_distribution_processed():
     return st.session_state[StateKeys.DATASET].plot_sampledistribution()
 
 
-# @st.cache_data  # TODO check if caching is sensible here and if so, reimplement with dataset-hash
-def get_display_matrix():
-    processed_df = pd.DataFrame(
-        st.session_state[StateKeys.DATASET].mat.values,
-        index=st.session_state[StateKeys.DATASET].mat.index.to_list(),
-    ).head(10)
-
-    return processed_df
-
-
 def display_matrix():
-    text = (
-        "Normalization: "
-        + str(
-            st.session_state[StateKeys.DATASET].preprocessing_info[
-                PreprocessingStateKeys.NORMALIZATION
-            ]
-        )
-        + ", Imputation: "
-        + str(
-            st.session_state[StateKeys.DATASET].preprocessing_info[
-                PreprocessingStateKeys.IMPUTATION
-            ]
-        )
-        + ", Log2-transformed: "
-        + str(
-            st.session_state[StateKeys.DATASET].preprocessing_info[
-                PreprocessingStateKeys.LOG2_TRANSFORMED
-            ]
-        )
-    )
-
     st.markdown("**DataFrame used for analysis** *preview*")
-    st.markdown(text)
 
-    df = get_display_matrix()
+    # TODO why not use the actual matrix here?
+    df = pd.DataFrame(
+        st.session_state[StateKeys.DATASET].mat.values,
+        index=st.session_state[StateKeys.DATASET].mat.index.to_list(),
+    ).head(10)
 
     st.dataframe(df)
 
-    show_button_download_df(df, file_name="analysis_matrix")
```

Comment:
Why did you remove the download button? I think it's crucial that the processed data can be downloaded.

Reply:
I felt it did not add too much value as only the first 10 rows were downloaded, but now I see that I should have solved that in a bit less destructive way :-D